CharlieBackend.Api/Middlewares/IsAccountActiveMiddleware.cs

```diff
@@ -36,55 +38,25 @@ public async Task InvokeAsync(HttpContext context, IAccountService accountServic
             }
             else
             {
-                string authHeader = context.Request.Headers["Authorization"];
+                var currentEmail = currentUserService.Email;
 
-                if (authHeader != null)
+                var isActive = await accountService.IsAccountActiveAsync(currentEmail);
+                
+                if (isActive == null)
                 {
-                    var handler = new JwtSecurityTokenHandler();
-                    authHeader = authHeader.ToString().Replace("Bearer ", "");
+                    context.Response.StatusCode = 401;
```

Comment:
@mahotskuy 

I'm not 100% sure there's a point in having this check: `if (isActive == null)` (line 45)
since it's impossible for not authenticated user to get past authentication middleware and access this middleware.

Here's current request pipeline configuration (`Startup.Configure()`):

![asdasdasd](https://user-images.githubusercontent.com/38729881/111172689-b573c900-85ae-11eb-99e3-b0f76a3fcfdd.png)

---

```suggestion
                    context.Response.StatusCode = StatusCodes.Status401Unauthorized;
```
Again, there are constants for that.

Reply:
Removed this redundant null check in https://github.com/ita-social-projects/WhatBackend/pull/372/commits/c6f2414908af5239dc0f3351fe157236ab4d9807